firmware/library/L2_Utilities/utility.cpp

```diff
@@ -0,0 +1,76 @@
+#include <cstdlib>
+
+#include "utility.hpp"
+
+uint32_t utility::Random(uint32_t min, uint32_t max)
```

Comment:
This should actually be `int32_t`. Users may want to accept negative numbers.

Reply:
Will do.